Lib/types.py

```diff
@@ -143,6 +143,37 @@ def _calculate_meta(meta, bases):
                         "of the metaclasses of all its bases")
     return winner
 
+
+def get_original_bases(cls, /):
+    """Return the class's "original" bases prior to modification by `__mro_entries__`.
+
+    This is useful for introspecting Generics.
+
```

Comment:
This sentence probably doesn't add much here, and it's good to keep the docstring as concise as possible

```suggestion
```